cmd/imagec/main.go

```diff
@@ -169,9 +184,78 @@ func main() {
 		if err := ic.PullImage(); err != nil {
 			log.Fatalf("Pulling image failed due to %s\n", err)
 		}
-	} else if imageCOptions.operation == PushImage {
+	case PushImage:
 		log.Errorf("The operation '%s' is not implemented\n", PushImage)
-	} else {
+	case ListLayers:
+		options := imageCOptions.options
```

Comment:
Minor: should we rename `imageCOptions` to `imagecOptions`? I don't think the `c` was ever meant to be capitalized. @caglar10ur do you recall?